style.css

```diff
@@ -69,3 +78,222 @@ body {
   left: 0;
   top: 472px;
 }
+
+.works {
+  display: grid;
+  grid-template-columns: auto;
+}
+
+.pro {
+  position: absolute;
+  width: 181px;
+  height: 50px;
+  left: 16px;
+  top: 660px;
+  font-family: 'Playfair Display', serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 50px;
+  line-height: 100%;
+  color: #1a2236;
+}
+
+.project {
+  display: flex;
+  flex-direction: column;
+}
+
+.pr-img1 {
+  width: 320px;
+  margin-left: 0;
+  margin-top: 780px;
+}
+
+.pr-img {
+  width: 320px;
+  margin-left: 0;
+  margin-top: 0;
+}
+
+.project-ins {
+  width: 320px;
+  margin-left: 0;
+  margin-top: 0;
+  background: #1a2236;
+  border: 0.333333px solid rgba(255, 255, 255, 0.1);
+  backdrop-filter: blur(0.5px);
+}
+
+.project-ins h3 {
+  width: 264px;
+  height: 16px;
+  margin-left: 20px;
+  margin-top: 20px;
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 300;
+  font-size: 20px;
+  line-height: 80%;
+  color: #fff;
+}
+
+.project-ins ul {
+  display: flex;
+  justify-content: start;
+  padding: 0;
+  margin-left: 16px;
+  gap: 16px;
+  margin-top: 0;
+  font-family: 'Poppins', sans-serif;
+  font-size: 10.3334px;
+  font-weight: 300;
+}
+
+.project-ins li {
+  background-color: #98dede;
+  list-style: none;
+  padding: 6px 10px;
+  text-align: center;
+}
+
+.project-ins button {
+  display: flex;
+  align-items: center;
+  width: 178px;
+  height: 44px;
+  margin-top: 20px;
+  margin-left: 17px;
+  margin-bottom: 20px;
+  background-color: #1a2236;
+  border: 1px solid #fff;
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 16px;
+  color: #fff;
+  gap: 15px;
+}
+
+a {
+  text-decoration: none;
+}
+
+.about {
+  display: flex;
+  flex-direction: column;
+}
+
+.about h2 {
+  width: 134px;
+  height: 50px;
+  margin-top: 56px;
+  margin-left: 16px;
+  margin-bottom: 0;
+  font-family: 'Playfair Display', serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 50px;
+}
+
+.about p {
+  width: 288px;
+  height: 126px;
+  margin-top: 20px;
+  margin-left: 16px;
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 14px;
+  color: #1a2236;
+}
```

Comment:
- [ ] According to the requirements and the original design, You must stick to the Figma design as much as possible, So I suggest to align this paragraph at left to match your Figma design 👍

> You can add `text-align: left` to fix this 💡

| Your Design  | Figma Design  |   
| --- | --- |
| <img src="https://user-images.githubusercontent.com/45231276/198961026-1e834db0-0bb7-4a83-990c-91f1477ec964.PNG" width="500" />  | <img src="https://user-images.githubusercontent.com/45231276/198961092-fee8ab4f-de4e-45f1-af93-593ca541de60.png" width="500" /> |